### DISCUSSION TITLE
Add the `"Message"` option in GitHub Profile

### DISCUSSION THREAD

**Discussion by ashikkumar23** [community/community#41811] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be nice to have a `"Message"` option in `GitHub` Profile, which would help to reach out to concerned people.

To cut down too many messages flooding in, maybe add an option to _message_ people being followed.

This would help to exchange ideas, share thoughts, etc., directly on the `GitHub` platform.

**Comment by jfullstackdev** [comment]
there are several requests for Direct Message on GitHub, I don't know whether the team is working on this one 

however, you have your Team Discussion as if it is a group chat when you join an Org with a team, 

what I think is being disallowed are random messages that you don't know, so if you just want a team discussion, GitHub already has that, in Discussion feature and Team Discussion in an Org
